DependencyInjection/CompilerPass/FixturesCompilerPass.php

```diff
@@ -30,8 +30,11 @@ public function process(ContainerBuilder $container)
         $definition = $container->getDefinition('doctrine.fixtures.loader');
         $taggedServices = $container->findTaggedServiceIds(self::FIXTURE_TAG);
 
+        $fixtures = array();
```

Comment:
Short array syntax please.

Reply:
Oops, I did this in purpose because I thought this project would require it. Will fix it :)

Reply:
Fixed